src/main/webapp/gameHelper.js

```diff
@@ -23,20 +23,31 @@ var experience;
 var threshold;
 // For now, this is hard coded, will change with new servlet
 // https://github.com/googleinterns/step144-2020/issues/171
-var newThreshold = 20;
- 
+var thresholdIncrement;
+
 function getDialogue() {
   const responsePromise = fetch('/game-dialogue');
   responsePromise.then(handleResponse);
 }
  
 function handleResponse(response) {
-  const textPromise = response.json();
+  const textPromise = response.text();
   textPromise.then(addDialogueToDom);
 }
  
-function addDialogueToDom(dialogue) {
+function addDialogueToDom(textResponse) {
+  responseArray = textResponse.split("\n");
+  // Divides the response into level and dialogue
+  var levelString = responseArray[0];
+  // Substring gets rid of the surrounding quotation marks in the string.
+  levelString = levelString.substring(1, levelString.length - 1);
+  thresholdIncrement = parseInt(levelString) * 5;
```

Comment:
Add a comment explaining that the threshold for the level should be set to 5 times the level of the game stage.